pkg/skaffold/docker/docker_init.go

```diff
@@ -34,7 +34,9 @@ var (
 )
 
 // Docker is the path to a dockerfile. Implements the InitBuilder interface.
-type Docker string
+type Docker struct {
+	Dockerfile string `json:"path"`
```

Comment:
nit: 
```suggestion
	File string `json:"path"`
```
to avoid stuttering